app/src/main/res/layout/camera.xml

```diff
@@ -14,23 +14,21 @@
             android:id="@+id/lnCam"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
-            android:gravity="center" >
+            android:layout_marginTop="16dp">
 
             <ImageButton
                 android:id="@+id/switch_camera"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_horizontal|center"
+                android:layout_width="20dp"
```

Comment:
@pa1pal @Abhi2424shek  i have now added the dimensions in dimen.xml and about setting them programtically, I think someone is working on layout for Camera UI, so we'll add the code in that PR.